builtin-functions/_functions.txt

```diff
@@ -1652,3 +1652,8 @@ class DateTimeImmutable implements DateTimeInterface {
 }
 
 function getenv(string $varname = '', bool $local_only = false): mixed;
+
+function yaml_emit_file ($filename ::: string, $data ::: mixed) ::: bool;
```

Comment:
`(string $filename, mixed $data)` and others, no :::, just type hints. Same for return — just ':' after ')'